src/components/Blocking/ReportUser.js

```diff
@@ -119,7 +119,7 @@ const styles = StyleSheet.create({
     marginBottom: 30
   },
   btnSkip: {
-    backgroundColor: COLORS.lightgrey,
+    backgroundColor: COLORS.gray100,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The background color has been changed from `COLORS.lightgrey` to `COLORS.gray100`. Please make sure that this color change is consistent across all similar components and doesn't affect the visibility of any overlaid text or icons.
```diff
-    backgroundColor: COLORS.lightgrey,
+    backgroundColor: COLORS.gray100,
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->